_posts/2022-09-26-changelog-179.markdown

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #179"
+date: 2022-09-26 13:00:00 +0300
+---
+
+This is the last release that supports the 2022.1 IDEs. To receive plugin updates further, please upgrade your IDE to 2022.2.
+
+## New Features
+
+* [#9241] Change the rendering of folded blocks for `use`, `mod`, `extern crate`, and `where` blocks to make them more distinct from folded block comments (by [@afetisov])
+
+  **TODO**: img/gif
+
+* [#9176] Use the [`--keep-going` Cargo option](https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#keep-going) when building and evaluating build scripts to continue building even if some crate fails to compile
+
+* [#9325] Enable [build script](https://doc.rust-lang.org/cargo/reference/build-scripts.html) evaluation in nightly plugin builds for 2022.2 platform
+
+* [#9043] Add [`redundant_semicolons`](https://doc.rust-lang.org/rustc/lints/listing/warn-by-default.html#redundant-semicolons) inspection and quick-fix for it
+
+  **TODO**: img/gif
+
+* [#7435] The intention to convert a match expression to a series of if let statements is now offered in more situations (by [@Kobzol])
```

Comment:
Unfortunately, I decided to [revert](https://github.com/intellij-rust/intellij-rust/pull/9396) this PR because of bug. So I think we don't need this item in this changelog